pom.xml

```diff
@@ -113,6 +113,12 @@
             <version>${lombok.version}</version>
             <scope>provided</scope>
         </dependency>
+        
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt</artifactId>
+            <version>0.9.1</version>
+        </dependency>
```

Comment:
@RulLu16 
First of all thank you for your job.

I have some questions bout this.
- why did you add the dependency to dss parent pom.xml not dss-core pom.xml?
- i read the document in https://github.com/jwtk/jjwt, the suggest using jjwt in general java project adding dependencies like: (it is different from yours)
``` xml
<dependency>
    <groupId>io.jsonwebtoken</groupId>
    <artifactId>jjwt-api</artifactId>
    <version>0.11.2</version>
</dependency>
<dependency>
    <groupId>io.jsonwebtoken</groupId>
    <artifactId>jjwt-impl</artifactId>
    <version>0.11.2</version>
    <scope>runtime</scope>
</dependency>
<dependency>
    <groupId>io.jsonwebtoken</groupId>
    <artifactId>jjwt-jackson</artifactId> <!-- or jjwt-gson if Gson is preferred -->
    <version>0.11.2</version>
    <scope>runtime</scope>
</dependency>
```

Reply:
In my local, there no error about dependency, so I didn't know about that. 
I'll fix this. Thank you!